src/main/java/com/cinema/Application.java

```diff
@@ -37,15 +41,29 @@ public static void main(String[] args) {
         session.setMovie(movie);
         session.setShowTime(LocalDateTime.now());
         sessionService.add(session);
-        sessionService.findAvailableSessions(movie.getId(),LocalDate.now())
+        sessionService.findAvailableSessions(movie.getId(), LocalDate.now())
                 .forEach(System.out::println);
 
         MovieSession sessionSecond = new MovieSession();
         sessionSecond.setCinemaHall(cinemaHall);
         sessionSecond.setMovie(movie);
-        sessionSecond.setShowTime(LocalDateTime.of(2020,11,1,12,0));
+        sessionSecond.setShowTime(LocalDateTime.of(2020, 11, 1, 12, 0));
         sessionService.add(sessionSecond);
-        sessionService.findAvailableSessions(movie.getId(),LocalDate.now())
+        sessionService.findAvailableSessions(movie.getId(), LocalDate.now())
                 .forEach(System.out::println);
+
+        User roma = new User();
+        roma.setEmail("matviichuk.office@gmail.com");
+        roma.setPassword("1234");
+        userService.add(roma);
+        User bot = new User();
+        bot.setEmail("viktor.pankov@gmail.com");
+        bot.setPassword("1234");
+        userService.add(bot);
+        System.out.println(userService.findByEmail(bot.getEmail()));
+        User newBot = new User();
+        newBot.setEmail("viktor.pankov@gmail.com");
+        newBot.setPassword("3214");
+        userService.add(newBot);
     }
```

Comment:
It would be nice to also test the register() and login() functions from the AuthenticationService)